README.md

```diff
@@ -15,11 +15,34 @@ file-per-thread-logger = "0.1.2"
 Then add this to your code, once per thread:
 
 ```rust
-file_per_thread_logger::initialize("file_prefix-");
+file_per_thread_logger::initialize("file_prefix-", None);
 ```
 
 Then each use of `log`'s primitive will log into files named the following way:
 - the main thread get a file that's suffixed after the program's name.
 - unnamed threads get a file suffixed with ThreadIdN where N is the thread's id
   number.
 - named threads get a file suffixed with the thread's name.
+
+#### Custom format for log messages
+
+You can customize the log messages by passing a custom format function.
+For example,
+```rust
+let formatter: file_per_thread_logger::FormatFn = |writer, record| {
```

Comment:
I have a slight preference to put this example in a doc comment, so that the tests won't pass if this has invalid syntax or gets out of sync with the code. Can you do this please?

Reply:
Of course. Would you like it in both places or just the doc comment?

Reply:
I think we could remove those in this README (so they don't fall out of sync) and link to docs.rs instead.